tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -1035,7 +607,7 @@ class ApiGwTests
             rr.stderr should include("does not exist")
         }
         finally {
-            val deleteresult = apiDeleteExperimental(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
+            val deleteresult = apiDelete(basepathOrApiName = testbasepath, expectedExitCode = DONTCARE_EXIT)
```

Comment:
a couple more "V2" related items
- testswaggerdoc1V2, testswaggerdoc2V2 
- WskPropsV2.. save this for last, but i think this can replace the existing WskProps definition in the same file.